src/server/actions/study-job-actions.ts

```diff
@@ -20,7 +21,20 @@ export const updateStudyJobStatusAction = async (info: MinimalJobInfo, status: S
         })
         .executeTakeFirstOrThrow()
 
+    if (results?.length) {
+        const blob = new Blob(results, { type: 'text/csv' })
+        const resultsFile = new File([blob], 'dummy_filename')
```

Comment:
Maybe this'll end up needing to be "results.csv" if this filename is what's exposed to the Researcher when they download